### PR TITLE
Added support for custom IP

### PR DESCRIPTION
**Pull request by ricardoboss**

### CODE REVIEW

File: Telemetry Viewer/src/CommunicationController.java [hunk 1]
```diff
@@ -2,11 +2,7 @@
 import java.io.IOException;
 import java.io.PipedInputStream;
 import java.io.PipedOutputStream;
-import java.net.DatagramPacket;
-import java.net.DatagramSocket;
-import java.net.ServerSocket;
-import java.net.Socket;
-import java.net.SocketTimeoutException;
+import java.net.*;
 import java.util.ArrayList;
```
Comment (hola-holo) [inline Telemetry Viewer/src/CommunicationController.java:6]: our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.

Comment (ricardoboss) [reply Telemetry Viewer/src/CommunicationController.java:11]: I added explicit imports again.

File: Telemetry Viewer/src/CommunicationController.java [hunk 2]
```diff
@@ -2,11 +2,7 @@
 import java.io.IOException;
 import java.io.PipedInputStream;
 import java.io.PipedOutputStream;
-import java.net.DatagramPacket;
-import java.net.DatagramSocket;
-import java.net.ServerSocket;
-import java.net.Socket;
-import java.net.SocketTimeoutException;
+import java.net.*;
```
Comment (zhouyinn) [inline Telemetry Viewer/src/CommunicationController.java]: Test for leaving a comment

Comment (zhouyinn) [inline Telemetry Viewer/src/CommunicationController.java]: Test for leaving a comment

**Comment by ricardoboss** [comment]
Is this still relevant?

**Comment by ricardoboss** [comment]
If no one needs this, I will close this PR. My branch will stay online though.

**Pull request closed without merging**